cdap-app-templates/cdap-etl/cdap-data-streams-base/src/main/java/io/cdap/cdap/datastreams/DataStreamsApp.java

```diff
@@ -52,4 +54,25 @@ public void configure() {
     }
     addSpark(new DataStreamsSparkLauncher(spec));
   }
+
+  @Override
+  public boolean isUpdateSupported() {
+    return true;
+  }
+
+  @Override
+  public ApplicationUpdateResult<DataStreamsConfig>
+  updateConfig(ApplicationUpdateContext applicationUpdateContext) throws Exception {
```

Comment:
nit: think it's a little more readable if the method name is on the previous line, with just the context on this line

Reply:
Done.